From 88d55c0533df1244d00822168bd5b22bd271b1c1 Mon Sep 17 00:00:00 2001
From: "guoyin.chen" <guoyin.chen@freescale.com>
Date: Mon, 19 Nov 2012 13:09:19 +0800
Subject: [PATCH 099/304] ENGR00234051 Enlarge the jvm heap size config

Repeatedly taking 5M picture from Camera App will make the heap
request exceed the 48M setting of heapgrowthlimit in previous config

Signed-off-by: guoyin.chen <guoyin.chen@freescale.com>
---
 imx6/imx6.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/imx6/imx6.mk b/imx6/imx6.mk
index fc53834..c3aca34 100644
--- a/imx6/imx6.mk
+++ b/imx6/imx6.mk
@@ -292,4 +292,4 @@ PRODUCT_DEFAULT_PROPERTY_OVERRIDES := \
 	persist.sys.usb.config=mtp
 
 # include a google recommand heap config file.
-include frameworks/native/build/tablet-dalvik-heap.mk
+include frameworks/native/build/tablet-7in-hdpi-1024-dalvik-heap.mk
-- 
1.8.0

